tests/installation/grub_test.pm

```diff
@@ -15,6 +15,11 @@ use utils;
 
 sub run() {
     my $self = shift;
+
+    if (get_var("SSH_KEY_IMPORT") || get_var("SSH_KEY_DO_NOT_IMPORT")) {
+        assert_screen 'inst-bootmenu';
+        send_key 'ret';
+    }
```

Comment:
please also add a description here, e.g. above line 19:
> In case of testing the import (or not-import) of ssh keys in installation, we are booting from the DVD, not HDD, so we need to select "Boot from HDD" here